### PR TITLE
Catty-4 synchronize english translation

### PR DESCRIPTION
**Pull request by KrennKristof**
Synchronized the english translations in the LanguageTranslationDefines with the Catroid [PR](https://github.com/Catrobat/Catroid/pull/3018/files), also made the "tapPlusToAdd" parameter-free and added the individual strings.

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catty/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#ios-general* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Comment by KrennKristof** [comment]
probably this is a good idea, that should be done for all of them, maybe I could do this in an extra task? because I guess there are multiple string tests in the UI-Tests.

another thing, I had a test that failed once, but only locally and I have no clue, why it failed that time, maybe this CI-Test run will fail too, even if I didn't change anything regarding this test.

**Comment by m-herold** [comment]
I think there's already a ticket for this. Let me check that. But just for the few strings you edited: please add the translation strings here to your PR directly.

**Comment by KrennKristof** [comment]
FYI: I tried different ways to squash the commits, it didn't work at all, I hope this doesn't affect the PR, the strings in the tests are now changed to the defined ones.

File: src/Catty UITests/Defines/LanguageTranslationDefinesUI.swift [hunk 1]
```diff
@@ -586,13 +590,13 @@ let kUIFEFunctionJoin = NSLocalizedString("join", bundle: Bundle(for: LanguageTr
...
 let kUIFEFunctionCeil = NSLocalizedString("ceil", bundle: Bundle(for: LanguageTranslation.self), comment: "")
-let kUIFEFunctionNumberOfItems = NSLocalizedString("number_of_items", bundle: Bundle(for: LanguageTranslation.self), comment: "")
+let kUIFEFunctionNumberOfItems = NSLocalizedString("number of items", bundle: Bundle(for: LanguageTranslation.self), comment: "")
 let kUIFEFunctionElement = NSLocalizedString("element", bundle: Bundle(for: LanguageTranslation.self), comment: "")
 let kUIFEFunctionContains = NSLocalizedString("contains", bundle: Bundle(for: LanguageTranslation.self), comment: "")
 
-let kUIFEFunctionScreenIsTouched = NSLocalizedString("screen_is_touched", bundle: Bundle(for: LanguageTranslation.self), comment: "")
-let kUIFEFunctionScreenTouchX = NSLocalizedString("screen_touch_x", bundle: Bundle(for: LanguageTranslation.self), comment: "")
-let kUIFEFunctionScreenTouchY = NSLocalizedString("screen_touch_y", bundle: Bundle(for: LanguageTranslation.self), comment: "")
+let kUIFEFunctionScreenIsTouched = NSLocalizedString("stage is tapped", bundle: Bundle(for: LanguageTranslation.self), comment: "")
```
Comment (m-herold) [inline src/Catty UITests/Defines/LanguageTranslationDefinesUI.swift]: I think this is called "touched" in Catroid (https://github.com/Catrobat/Catroid/pull/3018/files#diff-83cbece94e949621177890c9a504eec6R1747). Could you please check that. Thank a lot!

File: src/Catty/Defines/LanguageTranslationDefines.h [hunk 2]
```diff
@@ -586,13 +590,13 @@
...
 #define kUIFEFunctionCeil NSLocalizedString(@"ceil", nil)
-#define kUIFEFunctionNumberOfItems NSLocalizedString(@"number_of_items", nil)
+#define kUIFEFunctionNumberOfItems NSLocalizedString(@"number of items", nil)
 #define kUIFEFunctionElement NSLocalizedString(@"element", nil)
 #define kUIFEFunctionContains NSLocalizedString(@"contains", nil)
 
-#define kUIFEFunctionScreenIsTouched NSLocalizedString(@"screen_is_touched", nil)
-#define kUIFEFunctionScreenTouchX NSLocalizedString(@"screen_touch_x", nil)
-#define kUIFEFunctionScreenTouchY NSLocalizedString(@"screen_touch_y", nil)
+#define kUIFEFunctionScreenIsTouched NSLocalizedString(@"stage is tapped", nil)
```
Comment (m-herold) [inline src/Catty/Defines/LanguageTranslationDefines.h]: Probably the same here.

**Comment by m-herold** [comment]
Any news for this PR?

**Comment by KrennKristof** [comment]
not yet, Theo will make the PO Review

**Comment by GatCode** [comment]
Great work Kristof 👍🏻

**Pull request merged**